frontend/simple/views/NavBar.vue

```diff
@@ -10,27 +10,60 @@
         </div>
         <div class="nav-center">
           <!-- TODO: use v-for to dynamically generate these? -->
-          <router-link class="nav-item is-tab" active-class="is-active" to="new-group" id="CreateGroup"><i18n>Start a group</i18n></router-link>
-          <!-- <router-link class="nav-item is-tab" active-class="is-active" to="user" v-show="$store.state.loggedIn"><i18n>Profile</i18n></router-link> -->
-          <!-- <router-link class="nav-item is-tab" active-class="is-active" to="user-group" v-show="$store.state.loggedIn"><i18n>Group</i18n></router-link> -->
-          <router-link class="nav-item is-tab" active-class="is-active" to="pay-group" v-show="$store.state.loggedIn"><i18n>Pay Group</i18n></router-link>
-          <!-- <router-link class="nav-item is-tab" active-class="is-active" to="ejs-page" id="testEJS"><i18n>EJS test</i18n></router-link> -->
-          <router-link class="nav-item is-tab" active-class="is-active" to="event-log"><i18n>Event Log test</i18n></router-link>
-          <router-link id="MailboxLink" v-if="$store.state.loggedIn" class="nav-item" active-class ="is-active" to="mailbox">
-            <i18n>Mailbox</i18n>&nbsp;
-            <span id="AlertNotification" class="icon" style="color: #ed6c63" v-if="$store.getters.unreadMessageCount">
+          <router-link
+            active-class="is-active"
+            class="nav-item is-tab"
+            id="CreateGroup"
+            to="new-group"
+          >
+            <i18n>Start a group</i18n>
+          </router-link>
+          <router-link
+            active-class="is-active"
+            class="nav-item is-tab"
+            to="pay-group"
+            v-show="$store.state.loggedIn"
+          >
+            <i18n>Pay Group</i18n>
+          </router-link>
+          <router-link
+            active-class ="is-active"
+            class="nav-item"
+            id="MailboxLink"
+            to="mailbox"
+            v-if="$store.state.loggedIn"
+          >
+            <i18n>Mailbox</i18n>
+            <span
+              id="AlertNotification"
+              class="icon"
+              style="color: #ed6c63"
+              v-if="$store.getters.unreadMessageCount"
+            >
               <i class="fa fa-bell"></i>
             </span>
           </router-link>
         </div>
         <div class="nav-right">
           <span class="nav-item is-tab control">
             <router-link id="SignupBtn" v-show="!$store.state.loggedIn" class="button is-success" to="signup"><i18n>Sign Up</i18n></router-link>
-            <a href="#" id="LoginBtn" class="button"
-               v-bind:class="$store.state.loggedIn  ? 'is-danger' : 'is-primary'"
-              @click.prevent="toggleModal"
+            <a
+              class="button is-primary"
+              href="#"
+              id="LoginBtn"
+              v-if="!$store.state.loggedIn"
```

Comment:
This conditional is the exact same condition, just reversed, below. Use `v-else` then.